File: Voting_Machine/app/src/main/java/com/example/e_voteing_system/Helper/Date_helper.java
package com.example.e_voteing_system.Helper;

import com.google.type.DateTime;

import java.text.ParseException;
import java.text.SimpleDateFormat;
import java.util.Date;

public class Date_helper {
    public String convert_from_date_to_string(Date date){
        String date_string=date.toString();
        return date_string;
    }
    public Date convert_to_date_from_string(String string) throws ParseException {
        Date date;
        SimpleDateFormat simpleDateFormat=new SimpleDateFormat("dd-mm-yyyy");
        date=simpleDateFormat.parse(string);
        return date;
    }
    public boolean allow_to_vote(Date date){
        Date today=new Date();
        if((today.getYear()-date.getYear())>=18){
            return true;
        }
        else
            return false;
    }
}

File: Evoting_System_Website/myGP/Helper/Date_Helper.cs
//using Google.Type;
using System;
using System.Collections.Generic;
using System.Globalization;
using System.Linq;
using System.Web;


namespace myGP.Helper
{
    public class Date_Helper
    {
        public string ConvertDateToString(DateTime dt)
        {
            string date_string = dt.ToString();
            return date_string;
        }
        public DateTime ConvertToDate_FromString(string st)
        {
            DateTime d = DateTime.ParseExact(st, "dd/MM/yyyy", CultureInfo.InvariantCulture);
            return d;
        }
        public bool Check_Date(DateTime dt)
        {
            DateTime date = new DateTime();
            if ((date.Year - dt.Year) >= 18)
                return true;
            else
                return false;
        }
    }
}